tests/Fixer/Basic/BracesFixerTest.php

```diff
@@ -3029,14 +3029,7 @@ public function provideFixCommentBeforeBraceCases(): iterable
     } else {
         echo "unknown API";
         // sth
-    }
-
-    return $this->guess($class, $property, function (Constraint $constraint) use ($guesser) {
-        return $guesser->guessRequiredForConstraint($constraint);
-    // Fallback to false...
```

Comment:
This is yet another case of comment that is meant for the next line and thus should be unindented. This is not reliable and we should drop this IMO. Related discussion: #6490.

Reply:
#6490 is a bit different, as switch cases have no braces. This comment should just be indented with the rest of the closure body IMO. If it's meant for the next line, it should be placed before the next line, not inside another braces block:

```php
return $this->guess(
	$class,
	$property,
	function (Constraint $constraint) use ($guesser) {
		return $guesser->guessRequiredForConstraint($constraint);
	},
    // Fallback to false...
    // ... due to sth...
	false,
);
```